master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Felix Configuration Resource (FelixConfiguration)
+---
+
+A Felix configuration resource (FelixConfiguration) represents Felix configuration options for the cluster.
```

Comment:
This is another sentence that could be changed to be more active, specific, and interesting. Something more like "Use the Felix configuration resource to override any configuration settings configured through environment variables or a configuration file on a cluster-wide or per-host basis." I am not sure if my sentence is totally correct but perhaps it gives you a sense or some ideas.

Reply:
There is a more detailed explanation of this, that's why I added line 10 `See [Configuring Felix]({{site.baseurl}}/{{page.version}}/referance/felix/configuration) for more details.`  which explains these things in detail

Reply:
This may be the first sentence someone reads in their quest to understand what `Felix` is. Can we reference (or write) a high level summary of what Felix does here?